harness/determined/util.py

```diff
@@ -15,6 +15,13 @@
 import determined as det
 from determined_common import check, util
 
+try:
+    from tensorflow.python.framework.ops import EagerTensor
```

Comment:
I think we should probably just correct the framework-specific logic that results in this failure, rather than have some optional imports at this higher layer (which is supposed to be totally framework agnostic)

Reply:
Yeah I had noticed it was framework agnostic, but serializing Tensors seems like a reasonable thing for one common serialization function to do. There is similar logic in the _post_train_batch_end function that appears in the stack trace. So lemme see what I can do at that level. Withdrawing the PR for now.